connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/entities/ServerInfo.java

```diff
@@ -37,4 +39,9 @@ public String version() {
     public String commit() {
         return commit;
     }
+
+    @JsonProperty("kafka_cluster_id")
```

Comment:
If the clusterId is null, will the field still show up? I am wondering if there should be a descriptive sentinel instead?

Reply:
Makes sense.